coordinator/src/services/translation/prescription/index.ts

```diff
@@ -166,14 +166,10 @@ function convertDispensingSitePreference(
 function extractText(fhirCommunicationRequests: Array<fhir.CommunicationRequest>): Array<core.Text> {
   return fhirCommunicationRequests
     .flatMap(communicationRequest => communicationRequest.payload)
-    .map(contentString => new core.Text(contentString.contentString))
-}
-
-function isFirstRequestAndCommunicationRequestPresent(
-  request: number,
-  fhirCommunicationRequest: Array<fhir.CommunicationRequest>
-) {
-  return (request == 0 && fhirCommunicationRequest.length > 0)
+    .filter(Boolean)
+    .map(payload => payload.contentString)
```

Comment:
As mentioned above, should this be `.map(contentString => contentString.contentString)`? The double level naming is actually a little bit confusing, now that i look at it.

Reply:
Yeah maybe the class should be ContentStringPayload or something